python/looker_sdk/rtl/api_settings.py

```diff
@@ -146,7 +170,7 @@ def _override_from_env(self) -> Dict[str, str]:
 
         return overrides
 
-    def _clean_input(self, data: Dict[str, str]) -> Dict[str, str]:
+    def _clean_input(self, data: SettingsConfig) -> SettingsConfig:
```

Comment:
Reverted back to more or less the original version here with a `cast` at the end.

Reply:
works for me!